### PR TITLE
MGMT-2331 Host ignition customization

### PR DESCRIPTION
**Pull request by carbonin**
This builds on https://github.com/openshift/assisted-service/pull/466 to allow customizing the host pointer ignition in the same way we allow customizing the discovery ignition.

This PR adds a `POST` action on the `/clusters/{cluster_id}/hosts/{host_id}/ignition` endpoint which takes a payload that will be saved to the host row. This override is merged into the generated pointer ignition before it is uploaded to S3.

Fixes [MGMT-2331](https://issues.redhat.com/browse/MGMT-2331)

### CODE REVIEW

**Comment by carbonin** [comment]
This isn't really still in progress, but I'll wait until #466 is merged so this isn't so much to review.

I tested this with the changes from #466 as well as installer images built from https://github.com/openshift/assisted-installer/pull/42 and was able to place a file on the installed node.

**Review by filanov — APPROVED** [review]

**Pull request merged**
